### DISCUSSION TITLE
Not clear if you should use a Device or an Entity as a target [new user observation 6 of 7]

### DISCUSSION THREAD

**Discussion by davidbbs** [home-assistant/frontend#17231] [Automations / scripts / scenes]
I observed a new user creating simple light automations for three hours this weekend. 
Observation 6 of 7:

**Problem 6: Not clear if you should use a Device or an Entity as a target**
**Proposal 6: Hide one or the other**
_(Low-priority: the new user just always used entity, but did not know why.)_

It is not clear why you should choose a device or an entity when applying a service call. Both are available and most things are listed in both. If one is preferred, it should be shown and the other hidden.

**Comment by MatzeG1t** [comment]
i can confirm the uncertainty as quite a new user of HA. I got this comment here in a [thread](https://discord.com/channels/330944238910963714/1164644422227595305/1164669477212934205)

_"never use devices to automate off of, always the entities instead"_
So there seems to be a difference which could for example be mentioned here 
/config/entities
/config/devices/dashboard

With mention I mean a little "info" mark or a deep link to doc (which i didnt find right away right now)

For devices: 
_"A device is a model representing a physical or logical unit that contains entities."_ from this [page](https://www.home-assistant.io/getting-started/integration/).

For entities I didnt really found a explaination but how I understand it, they are concepts/properties/attributes from devices that can be read/written.

This would help to improve the learnability.

WHen it comes to automations and using on or the other: most dialogues allow one or the other and not both at the same time right?
